boards/arm/mikroe_mini_m4_for_stm32/mikroe_mini_m4_for_stm32.dts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019, Kwon Tae-young <tykwon@m2i.co.kr>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f415Rg.dtsi>
+
+/ {
+	model = "STMicroelectronics Mikroe MINI-M4 for STM32 board";
+	compatible = "st,stm32f415-mikroe-f415rg", "st,stm32f415";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,ccm = &ccm0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		orange_led_1: led_1 {
+			gpios = <&gpioc 12 GPIO_INT_ACTIVE_HIGH>;
+			label = "DATA LD1";
+		};
+		red_led_2: led_2 {
+			gpios = <&gpioc 13 GPIO_INT_ACTIVE_HIGH>;
+			label = "STAT LD2";
+		};
+	};
+
+	aliases {
+		led0 = &orange_led_1;
+		led1 = &red_led_2;
+	};
+};
+
```

Comment:
Any reason not adding I2C/SPI/USB as described in board pinout doc page?

Reply:
I saw the phrase below at #9333.
So i did not add I2C/SPI/USB.
> -Limit initial work to the basics drivers required for hello_world and blinky: clock_control, uart, exti, gpio, pinmux

I2C/SPI/USB/PWM have been updated.

Reply:
Actually comment in #9333 was for a full series introduction which is much more work than a board introduction. Also when the work is quite good at first time, we can do more :-)